metadata.json

```diff
@@ -22,7 +22,7 @@
     },
     {
       "name": "theforeman/foreman_proxy",
-      "version_requirement": ">= 13.0.0 < 17.0.0"
+      "version_requirement": ">= 13.0.0 < 18.0.0"
```

Comment:
I prefer these to at least be in separate commits. Releasing and allowing a new version are separate actions. You may even want to cherry pick allowing a newer version but you really don't cherry pick releases.